### DISCUSSION TITLE
Feb 8

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#132] [DailyMeeting]
### ⚽️ 회의 내용

**개발 현황**
달리는 중 🏃🏃‍♂️🏃‍♀️

**루틴**
수: 루틴 등록되지 않은 경우 -> 그날 운동 기록 작성 -> 해당 요일 루틴으로 지정
  or 캘린더에서 선택한 날짜의 운동 기록을 불러오고, 수정
수영: 요일마다 지정한 루틴 불러오기
윤혁: 기본 기능으로 넣기는 그렇다. 캘린더에서 선택한 날짜의 운동 기록을 불러오고, 수정 >> 맘에 듦
> 2시에 미팅 참여해서 말해보자

**피그마**
공통 컴포넌트 분류 (개발완료 여부)

**운동기록**
하나의 폴더에 페이지 2개

**운동 변수명**
exercise

**jotai로 빼는 기준**
depth 2까지만....
이후는 전역으로 처리하

### 🏀 오늘 할 일

 슬희: 로그인 페이지 합치고 회원가입 진행할 예정
 수: 운동 기록 계속
 수영: axios, 에러 처리 로직, 지도
 윤혁: 마이페이지 디자인

### 🎱 TMI

슬희: 4시반에 결과 나옴.. 근데 안 좋을까봐 걱정 ..... 목 초음파에서 이상한 거 나옴 ....
수: 일본 여행 숙소 취소 당함... 호스트가 잠수 탐 ~~!! 
수영: 쥬리,, 설 기념 꼬까옷 맞춤~ 
윤혁: 바지를 샀는데 맘에 쏙~ 👖 배기진~

**Comment by oauch** [comment]
벌룬핏 바지였네요 🙄

**Comment by hanseulhee** [comment]
쥬리포시 납치해도 돼여ㅕ?
